### ISSUE TITLE
vm boot slow for arch linux arm64

### ISSUE THREAD

**Issue by Magniquick** [lima-vm/lima#858]
### Description

Was just trying out the inbuilt arch linux vm config when I found it to have a slow bootup compared to the default config
digging into the logs, It hangs on things like -
``>>Start HTTP Boot over IPv4InstallProtocolInterface: BA23B311-343D-11E6-9185-5820B1D65299 13D12B178``
I tried enabling the coca ui for qemu and removing / changing the priority of those boot options, but it looks they do not persist across reboots.
[This](https://askubuntu.com/a/192139) also doesn't work  
Strangely enough, this doesn't happen with the ``default`` instance

limactl version 0.10.0
qemu: 6.2.0
OS: macOS 12.3.1 21E258 arm64

**Comment by AkihiroSuda** [comment]
Cc @mcginty

**Comment by mcginty** [comment]
Thanks for opening the issue. I'll look into the boot options and see if we can speed it up.

**Comment by Magniquick** [comment]
Here is a comparison - 
```limactl start default  0.01s user 0.02s system 0% cpu 23.402 total```
```limactl start archlinux  0.03s user 0.04s system 0% cpu 4:26.07 total```
That's a slowdown of _11 times_, if I am right.

**Comment by Magniquick** [comment]
@mcginty sorry for troubling you, is there any update on the issue ?

**Comment by Magniquick** [comment]
@AkihiroSuda Is there any way to modify the qemu xml files ? or pass commandline options to qemu ?

**Comment by mcginty** [comment]
@Magniquick sorry, I hadn't looked at it yet. Will take a peek today.

**Comment by mcginty** [comment]
Yeah, I think the problem is that I used the "cheapest" method for booting with qemu's UEFI which is [using a `startup.nsh` script](https://wiki.archlinux.org/title/EFISTUB#Using_a_startup.nsh_script). As I understand it, this boots into the UEFI Shell first rather than directly into linux, then the shell tries things and decides eventually to look at (for us) `/boot/startup.nsh` and try that out too.

**Comment by Magniquick** [comment]
> Yeah, I think the problem is that I used the "cheapest" method for booting with qemu's UEFI which is [using a `startup.nsh` script](https://wiki.archlinux.org/title/EFISTUB#Using_a_startup.nsh_script). As I understand it, this boots into the UEFI Shell first rather than directly into linux, then the shell tries things and decides eventually to look at (for us) `/boot/startup.nsh` and try that out too.

so you just need to copy defaults startup method ?

**Comment by Magniquick** [comment]
@mcginty Sorry for the ping, butis there any update on this ?
